src/main/scala/ScalaSpace.scala

```diff
@@ -0,0 +1,82 @@
+package scalaspace
+
+import google.maps._
+import org.scalajs.dom._
+import upickle.default._
+
+import scala.scalajs.js
+import scala.scalajs.js.JSApp
+import scala.scalajs.js.annotation.JSExport
+
+object ScalaSpace extends JSApp {
+
+  val infoWindow = new InfoWindow()
+
+  def onClick(map: Map, marker: Marker, group: Group) = () => js.Function {
+    val link = if (group.url.startsWith("http")) group.url else s"http://www.meetup.com/${group.url}/"
+    val content = document.createElement("a")
+    content.setAttribute("href", link)
+    content.innerHTML = group.name
+    infoWindow.setContent(content)
+    infoWindow.open(map, marker)
+    ""
+  }
+
+  def logo(group: Group): Icon =
+    if (group.justScala) Icon("img/markers/scala.png") else Icon("img/markers/lambda.png")
+
+  def initialize() = js.Function {
```

Comment:
Same idea here:
```scala
def initialize(): Unit = {
  ...
}
```
and then
```scala
google.maps.event.addDomListener(window, "load", () => initialize())
```
or, in this case you can also do this if you prefer:
```
google.maps.event.addDomListener(window, "load", initialize _)
```
but I find the former version more readable most of the time.